Views/GIDiffViewController.m

```diff
@@ -79,8 +79,8 @@ - (void)setCommit:(GCCommit*)commit withParentCommit:(GCCommit*)parentCommit {
       [_diffContentsViewController setDeltas:diff.deltas usingConflicts:nil];
       [_diffFilesViewController setDeltas:diff.deltas usingConflicts:nil];
       
-      _fromTextField.stringValue = [NSString stringWithFormat:@"\"%@\" <%@> (%@)", _commit.summary, _commit.shortSHA1, [_dateFormatter stringFromDate:_commit.date]];
```

Comment:
This should be changed into NSLocalizedString() in a separate and prior commit to make it clear it's a UI string.